run/john.conf

```diff
@@ -3497,6 +3497,42 @@ void restore()
 	}
 }
 
+# generate all possible wps pins
+[List.External:wpspin]
+int pin;
+void init() {
+	pin = 0;
+}
+void generate() {
+	if(pin > 9999999) {
+		word[0] = 0;
```

Comment:
Can do `word = 0;`, like we use in other external modes.

Reply:
thanks for review. i took this approach from external:case. updated.